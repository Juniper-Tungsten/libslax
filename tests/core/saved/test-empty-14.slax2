version 1.2;

ns foo exclude = "http://example.com/foo";
ns jcs extension = "http://xml.juniper.net/jcs";
ns a = "http://example.com/1";
ns "http://example.com/global";
ns b = "http://example.com/2";

main {
    ns c = "http://example.com/3";
    
    <top> {
        ns a = "http://example.com/4";
        
        apply-templates commit-script-input/configuration;
    }
}
