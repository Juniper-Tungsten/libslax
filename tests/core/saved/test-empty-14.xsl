<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:foo="http://example.com/foo" xmlns:jcs="http://xml.juniper.net/jcs" xmlns:a="http://example.com/1" xmlns="http://example.com/global" xmlns:b="http://example.com/2" version="1.0" exclude-result-prefixes="foo" extension-element-prefixes="jcs">
  <xsl:template xmlns:c="http://example.com/3" match="/">
    <top xmlns:a="http://example.com/4">
      <xsl:apply-templates select="commit-script-input/configuration"/>
    </top>
  </xsl:template>
</xsl:stylesheet>
