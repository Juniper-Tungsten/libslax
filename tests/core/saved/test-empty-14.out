<?xml version="1.0"?>
<top xmlns:a="http://example.com/4" xmlns="http://example.com/global" xmlns:c="http://example.com/3" xmlns:b="http://example.com/2"/>
